python/lsst/ts/cbp/csc.py

```diff
@@ -78,6 +78,12 @@ async def do_move(self, data):
             self.component.move_elevation(data.elevation),
             self.component.move_azimuth(data.azimuth),
         )
+        # self.log.debug("Moving Elevation from CSC move command")
```

Comment:
Is there a reason why these lines are commented out and why to keep them? If yes, please add a comment that justifies why they need to be kept and otherwise please remove them.